acquisitions/src/main/resources/thunderjet/cross-modules/features/rollover-with-no-settings.feature

```diff
@@ -0,0 +1,162 @@
+# For MODORDERS-1162
+Feature: Rollover with no settings
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * call loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json, text/plain' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json, text/plain' }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def createFiscalYear = read('classpath:thunderjet/mod-finance/reusable/createFiscalYear.feature')
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+    * def openOrder = read('classpath:thunderjet/mod-orders/reusable/open-order.feature')
+    * def validateOrderLineEncumbranceLinks = read('classpath:thunderjet/mod-orders/reusable/validate-order-line-encumbrance-links.feature')
```

Comment:
I think these declarations is not need since they have been already set to karate.json file
https://github.com/folio-org/folio-integration-tests/blob/d4589f5d9c45aab54d9f55393bbd19eefe413481/acquisitions/src/main/resources/karate-config.js#L62-L64

we can add validateOrderLineEncumbranceLinks if it is being used frequently in subseqent features